app/Helpers/helpers.php

```diff
@@ -2,17 +2,23 @@
 
 function bannerImg()
 {
-    $context = stream_context_create(['http' => ['header' => 'Connection: close\r\n']]);
-    $url = 'https://viblo.asia/';
-    $start = config('view.start');
-    $matchLeng = config('view.maxBanner');
-    $content = file_get_contents($url, false, $context, $start, $matchLeng);
-    $data = preg_match('/<div class="container px-0"><img src="(.*?)"/is', $content, $match);
-    if (isset($match[1])) {
-        return $match[1];
-    } else {
-        return null;
+    try {
+        $context = stream_context_create(['http' => ['header' => 'Connection: close\r\n']]);
+        $url = 'https://viblo.asia/';
+        $start = config('view.start');
+        $matchLeng = config('view.maxBanner');
+        $content = file_get_contents($url, false, $context, $start, $matchLeng);
+        $data = preg_match('/<div class="container px-0"><img src="(.*?)"/is', $content, $match);
+        if (isset($match[1])) {
+            return $match[1];
+        } else {
+            $match[1] = 'https://viblo.asia/announcements/gioi-thieu-state-of-chain-tieu-diem-hackathon-meet-up-blockchain-nam-2018-YWOZrpmp5Q0';
+            return $match[1];
+        }
+    } catch (Exception $e) {
+        return view('error');
     }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Helpers/helpers.php</br>**Line**: 22</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace